tests/test_jmcomic/test_jm_async_api.py

```
@@ -0,0 +1,117 @@
+"""
+Async API 行为一致性测试 (参考 test_jm_api.py)
+
+测试 download_album_async / download_photo_async / download_batch_async 的行为，
+确保与 sync 版 download_album / download_photo / download_batch 行为一致。
+"""
+
+from test_jmcomic import *
```

Ruff (0.15.15)

[error] `from test_jmcomic import *` used; unable to detect undefined names

(F403)